lib/tpl/dokuwiki/css/_admin.css

```diff
@@ -1,62 +0,0 @@
-/**
- * This file provides styles for the Administration overview
- * (?do=admin).
- */
-
-.dokuwiki ul.admin_tasks {
-    float: left;
-    width: 40%;
-    list-style-type: none;
-    font-size: 1.125em;
-}
-[dir=rtl] .dokuwiki ul.admin_tasks {
-    float: right;
-}
-
-.dokuwiki ul.admin_tasks li {
-    padding-left: 35px;
-    margin: 0 0 1em 0;
-    font-weight: bold;
-    list-style-type: none;
-    background: transparent none no-repeat scroll 0 0;
-    color: inherit;
-}
-[dir=rtl] .dokuwiki ul.admin_tasks li {
-    padding-left: 0;
-    padding-right: 35px;
-    background-position: right 0;
-}
-
-.dokuwiki ul.admin_tasks li.admin_acl {
-    background-image: url(../../images/admin/acl.png);
```

Comment:
These images should be deprecated instead of removing them. Some templates still use them.